accepted/future-releases/static-extension-methods/feature-specification.md

```diff
@@ -323,6 +325,8 @@ The syntax is not *convenient*&mdash;you have to put the "constructor" invocatio
 
 An expression of the form `MyList(object)` or `MyList<String>(object)` must *only* be used for extension member access. It is a compile-time error to use it in any other way, similarly to how it is a compile-time error to use a *prefix* for anything other than member access. This also means that you cannot use an override expression as the receiver of a cascade, because a cascade does evaluate its receiver to a value. Unlike a prefix, it doesn't have to be followed by a `.` because extensions can also declare operators, but it must be followed by a `.`, a declared operator, or an arguments part (in case the extension implements `call`).
 
+If an extension override expression is the first operand of a `==` operator, then the evaluation of that operator does *not* check whether the operands are `null` before invoking the `==` method of the extension. It is treated as a completely normal static method invocation. Checking for `null` before invoking an `operator==` implementation is reserved for *instance* members, not static extension members.
```

Comment:
Is `MyExt(object) != whatever` allowed?  Presumably it's an error if the return type of `MyExt.==` is not bool?